src/components/Header/HeaderButtons/index.tsx

```diff
@@ -116,82 +119,67 @@ const HeaderButtons = ({ setIsChooseNetwork }: IHeaderButtonsProps) => {
       setCurrentSubgraphInfo({
         network: chainInfo.network,
         status: SubgraphStatus.FetchingData,
-        chainIcon: chainInfo.icon
+        chainIcon: chainInfo.icon,
+        diffTime: '0'
       })
 
       return
     }
 
-    const chaindId = Number(wallet?.chains[0].id ?? 0)
-    const arbitrumChainId = 42161
-    if (chaindId === arbitrumChainId) {
-      const provider = new JsonRpcProvider(networks[chaindId].rpc)
-
-      const subgraphTimestamp = await provider.getBlock(
-        BigInt(latestBlockData?.subgraphBlock)
-      )
-      const currentTimestamp = await provider.getBlock(
-        BigInt(latestBlockData?.currentBlock)
-      )
-
-      if (!(currentTimestamp?.timestamp && subgraphTimestamp?.timestamp)) {
-        return setCurrentSubgraphInfo({
-          network: chainInfo.network,
-          status: SubgraphStatus.FetchingData,
-          chainIcon: chainInfo.icon
-        })
-      }
-
-      //The return of getBlock from ethers is with timestamps in seconds.
-      const timestampDiffInSeconds = Math.abs(
-        currentTimestamp.timestamp - subgraphTimestamp.timestamp
-      )
-      const diffInMinutes = Math.floor(timestampDiffInSeconds / 60)
-
-      const fiveMinutes = 5
-      const sixHoursInMinutes = 360
-      if (diffInMinutes <= fiveMinutes) {
-        return setCurrentSubgraphInfo({
-          network: chainInfo.network,
-          status: SubgraphStatus.Updated,
-          chainIcon: chainInfo.icon
-        })
-      }
-      if (diffInMinutes <= sixHoursInMinutes) {
-        return setCurrentSubgraphInfo({
-          network: chainInfo.network,
-          status: SubgraphStatus.PracticallyUpdated,
-          chainIcon: chainInfo.icon
-        })
-      }
+    const chaindId =
+      networks[Number(wallet?.chains[0].id ?? '')]?.chainId ?? 43114
 
-      setCurrentSubgraphInfo({
+    const provider = new JsonRpcProvider(networks[chaindId].rpc)
+    const subgraphTimestamp = await provider.getBlock(
+      BigInt(latestBlockData?.subgraphBlock)
+    )
+    const currentTimestamp = await provider.getBlock(
+      BigInt(latestBlockData?.currentBlock)
+    )
+
+    if (!(currentTimestamp?.timestamp && subgraphTimestamp?.timestamp)) {
+      return setCurrentSubgraphInfo({
         network: chainInfo.network,
-        status: SubgraphStatus.Outdated,
-        chainIcon: chainInfo.icon
+        status: SubgraphStatus.FetchingData,
+        chainIcon: chainInfo.icon,
+        diffTime: '0'
       })
     }
 
-    const blocksInFiveMinutes = 150
-    const blocksInSixHours = 10800
-    if (latestBlockData.diff <= blocksInFiveMinutes) {
+    const dataDiff = getDateDiff(
```

Comment:
"date" instead of "data"

```suggestion
    const dateDiff = getDateDiff(
```